### PR TITLE
Add non-interactive mode

### PR DESCRIPTION
**Pull request by roman-mazur**
Can be used to run the client from CI jobs.

Change-type: minor
Signed-off-by: Roman Mazur <roman@balena.io>

### CODE REVIEW

**Comment by roman-mazur** [comment]
@balena-ci retest please

**Comment by roman-mazur** [comment]
@balena-ci retest please

File: client/bin/multi-client.js [hunk 1]
```diff
@@ -38,10 +39,51 @@ const yargs = require('yargs')
 		type: 'boolean',
 		default: false,
 	})
+	.option('n', {
+		alias: 'none-interactive',
```
Comment (telphan) [inline client/bin/multi-client.js]: NIT: non-interactive not none-interactive

Comment (roman-mazur) [reply client/bin/multi-client.js]: thanks

**Comment by roman-mazur** [comment]
@balena-ci retest please

**Pull request merged**